### DISCUSSION TITLE
Use of the analog inputs

### DISCUSSION THREAD

**Discussion by 0lek** [micro-nova/AmpliPi#291] [General]
Hey guys,

This is a very open question: what do you use / plan to use the RCA inputs for? I somehow feel that I won't have a real use for them - I mostly stream or liste to internet radio... I don't have a cd player to connect there. Maybe a roon bridge some time in the future?

Just curious of what other people's use cases are.

Cheers,
Aleksander

**Comment by sepffuzzball** [comment]
That's funny, because I totally am using all 4 of my RCA inputs at the moment!

1. Currently hooked up to the spouse's gaming PC as an input which is in the same server rack (their office is in the room right next to it, so we just run the cables accordingly) - normally it just outputs to the tower speakers that are in the office but gives the flexibility that they can play a youtube video or something and have their PC audio output to any other room in the house
2. Currently hooked up to my gaming PC in the upstairs office via RCA-over-ethernet adapters for...pretty much the same usecase as the spouse's gaming PC
3. Currently hooked up to an Intel NUC in the server rack that runs Home Assistant for VLC_telnet so we can easily play audio automations and use it for TTS. I suspect this may change at some point, but it's also nice knowing I've got an always-on way to play something
4. Currently hooked up to a Chromecast Audio for the Android phones in the house. We have 4 people in the house, two iOS, two Android and we all listen to music differently. 1 iOS and 1 Android user uses Spotify regularly so Spotify connect works great. 1 other iOS user typically uses Airplay. 1 Android user uses Youtube Music/Youtube so having a cast target for them makes sense

**Comment by bs42** [comment]
Currently I'm using Mopidy on my NUC that also runs Home Assistant using one of the RCA, until Mopidy can be integrated onto the AmpliPi itself, it allows me to intermix Spotify and local music (flac).

I'm in the process of wiring up another of the RCA plugs using RCA over ethernet baluns to push the main bedroom's TV audio over that rooms speakers (plus the attached bathroom's speakers) instead of the builtin TV speakers.

**Comment by 0lek** [comment]
Interesting. Thanks for the suggestions. We're all Apple and so far spotify conenct works greatly. I could image hooking up home assistant but don't see that many potential for "audio automations" as of yet. But then again I use home assistant only as a visualization platform and not the "brain". But could potentially be interesting as a backup.
